backend-rust/src/indexer.rs

```diff
@@ -1810,14 +1844,18 @@ struct PreparedContractInitialized {
     name:             String,
     amount:           i64,
     height:           i64,
+    cis2_events:      Vec<cis2::Event>,
 }
 
 impl PreparedContractInitialized {
-    fn prepare(
+    async fn prepare(
+        node_client: &mut v2::Client,
+        pool: &PgPool,
```

Comment:
We should not access the database in the preprocessing phase, but it seems this is not used anyway.
Please remove